app/views/postquotes/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_for [@postquote] do |f| %>
```

Comment:
This doesn't need to be an array if there's only one of them. @postquote is fine rather than [@postquote]